src/xdocs/checks.xml

```diff
@@ -597,6 +597,10 @@
         <td>
         Checks line wrapping for operators.</td>
       </tr>
+      <tr>
+        <td><a href="config_misc.html#OrderedProperties">OrderedProperties</a></td>
+        <td>Detects if keys in properties files are in correct order.</td>
```

Comment:
Please synchronize all your descriptions of the check to be the same. (Java, xdoc, checks)